packages/core/src/types.ts

```diff
@@ -237,9 +239,12 @@ export interface CookiesOptions {
   csrfToken: CookieOption
   pkceCodeVerifier: CookieOption
   state: CookieOption
+  signinInfo: CookieOption
   nonce: CookieOption
 }
 
+export type SigninInfo = string
```

Comment:
Is it `SigninInfo` or `SignInInfo` 🤔🤔 Choosing the former arbitrarily because less stuttering.